recipes/libiginition-math/bld.bat

```diff
@@ -0,0 +1,14 @@
+mkdir build
```

Comment:
The directory name of this recipe has a typo, `libiginition-math` instead of `libigniition-math`.

Reply:
Good catch! Fixed it now.